libcudacxx/include/cuda/std/detail/libcxx/include/__utility/swap.h

```diff
@@ -30,19 +30,19 @@
 _LIBCUDACXX_BEGIN_NAMESPACE_STD
 
 template <class _Tp>
-inline _LIBCUDACXX_INLINE_VISIBILITY
-_LIBCUDACXX_CONSTEXPR_AFTER_CXX11 __swap_result_t<_Tp>
-swap(_Tp& __x, _Tp& __y) _NOEXCEPT_(is_nothrow_move_constructible<_Tp>::value &&
```

Comment:
Why is it now safe to use `noexcept` directly?

Reply:
We have check that searches for 
```cpp
#if !(__has_feature(cxx_noexcept))
#define _LIBCUDACXX_HAS_NO_NOEXCEPT
#endif
```

This is true for all compilers we support so I am in the habit of phasing it out whenever I modernize some code

Shout if you would prefer one fell swoop